src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = sourceString.split(';').map((el) => el.split(':'));
+
+  const result2 = {};
```

Comment:
[CODE STYLE] - The variable name `result2` is also too generic and does not convey the content it holds. Consider renaming it to something more descriptive like `styleObject`.